### ISSUE TITLE
Solana support added but not for its tokens

### ISSUE THREAD

**Issue by eribertomota** [LedgerHQ/ledger-live-desktop#4839]
Thanks to Ledger-Live team for adding Solana (partially).

Currently, I can see my Sol in Accounts, but the tokens as USDC aren't shown. I can see all tokens using Solflare.

Is there a prevision to add support for Solana tokens?

Thanks again.

**Comment by heavenly** [comment]
yes, also can we get support for an NFT view like how the ETH accounts have it?
